### DISCUSSION TITLE
NPM

### DISCUSSION THREAD

**Discussion by giscus** [padosum/blog#62] [General]
# NPM

NPM (Node Package Manager)    명령어로 자바스크립트 라이브러리를 설치하고 관리할 수 있는 [ Package-Manager|패키지 매니저 ], 전 세계 자바스크립트 개발자들이 자바스크립트 라이브러리를 공개된 저장소에 올려놓고  …

https://www.padosum.dev/wiki/NPM/

**Comment by youngkyo0504** [comment]
padosum님 잘지내시나요
npm ci가 2배나 빠른거 신기하네용

**Reply by padosum** [reply]
ㅋㅋ 전 잘지내지요 새해 복 많이받으시고.. 올해도 뵐 수 있음 좋겠네요 !

**Reply by youngkyo0504** [reply]
서울 올라오시면 연락주십쇼 ㅎㅎ 다 끌어 모으겠습니다.
